include/util/static_rtree.hpp

```diff
@@ -73,7 +76,9 @@ class StaticRTree
         LeafNode() : object_count(0), objects() {}
         uint32_t object_count;
         std::array<EdgeDataT, LEAF_NODE_SIZE> objects;
+        unsigned char __alignment[LEAF_PAGE_SIZE - sizeof(uint32_t) - sizeof(EdgeDataT) * LEAF_NODE_SIZE];
```

Comment:
Could you change the naming for `__alignment`. Symbols starting with `_ _` are reserved for the stdlib implementation and I feel like "alignment" is generic enough that there is the probability for a clash somewhere somehow.